CHANGELOG.md

```diff
@@ -4,7 +4,22 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## 0.2.0 - 2017-10-20
+### Added
+- Automatically resolve main class in launching. See [Issue#9](https://github.com/Microsoft/vscode-java-debug/issues/9)
+- Fully support external source files together with changes from VSCode. See [PR#58](https://github.com/Microsoft/java-debug/pull/58)
+
+### Changed
+- Adopt the new VSCode Debug protocol. See [PR#87](https://github.com/Microsoft/vscode-java-debug/pull/87)
+- Display stackframe name with format ClassName.MethodName(ParameterList). See [PR#73](https://github.com/Microsoft/java-debug/pull/73)
```

Comment:
Display function names in the format of ClassName.MethodName(Parameter List...).